### DISCUSSION TITLE
call frontend service failed

### DISCUSSION THREAD

**Discussion by lzzcj** [apache/doris#15523] [A - General / Q&A]
doris版本1.1.1
执行datax写doris时报call frontend service failed的错误，之前有改过fe.conf的配置然后重启过。有人知道是什么原因吗？
be日志如下：
W1102 15:18:44.785388 17134 fragment_mgr.cpp:241] Got error while opening fragment 254b3a6c50de4361-97f4a9f86f262ef2: Cancelled: Cancelled
W1102 16:46:47.238984 17109 vdata_stream_sender.h:249] failed to send brpc batch, error=RPC call is timed out, error_text=[E1008]Reached timeout=300000ms @10.55.1.108:8060, client: 10.55.1.107
W1102 16:46:47.239799 17109 fragment_mgr.cpp:241] Got error while opening fragment 20526df6a1394842-92b4b7a07f0b86b6: Thrift rpc error: failed to send brpc batch, error=RPC call is timed out, error_text=[E1008]Reached timeout=300000ms @10.55.1.108:8060, client: 10.55.1.107
W1102 16:46:47.254287 17109 vdata_stream_sender.h:249] failed to send brpc batch, error=RPC call is timed out, error_text=[E1008]Reached timeout=300000ms @10.55.1.108:8060, client: 10.55.1.107
W1102 16:46:47.254333 17109 vdata_stream_sender.h:249] failed to send brpc batch, error=RPC call is timed out, error_text=[E1008]Reached timeout=300000ms @10.55.1.108:8060, client: 10.55.1.107
W1103 15:30:01.428149 17203 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.429009 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.429538 17203 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.430187 17202 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.430976 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.431612 17202 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.432267 17203 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.433130 17202 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.433645 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.434274 17203 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 16:01:42.258661 17156 tablet.cpp:541] delete stale rowset tablet=49082.1505079904.314f77fcf9c8a8bc-a8904b682d00949d version[0,1] not find in stale rs version map
W1103 16:04:43.331810 17156 tablet.cpp:541] delete stale rowset tablet=49078.1505079904.97469d0d0e1463c9-7c82eddec3a58fa7 version[0,1] not find in stale rs version map
W1103 16:04:43.332846 17156 tablet.cpp:541] delete stale rowset tablet=49086.1505079904.d14c77805f0094fd-8cace9c0e85f0c8e version[0,1] not find in stale rs version map
W1103 16:04:43.334069 17156 tablet.cpp:541] delete stale rowset tablet=49110.1505079904.bc4062d1c6bb7cc3-3f769aafca3c6bbe version[0,1] not find in stale rs version map
W1103 16:04:43.334560 17156 tablet.cpp:541] delete stale rowset tablet=49106.1505079904.dc497bf3d4c5746b-5641ff032295a6bf version[0,1] not find in stale rs version map
W1103 16:04:43.335085 17156 tablet.cpp:541] delete stale rowset tablet=49102.1505079904.2247653a206d9982-eb0264d8be654990 version[0,1] not find in stale rs version map
W1103 16:04:43.335626 17156 tablet.cpp:541] delete stale rowset tablet=49098.1505079904.2a44367aa101c75a-ebbc78aa590b04b3 version[0,1] not find in stale rs version map
W1103 16:04:43.336102 17156 tablet.cpp:541] delete stale rowset tablet=49094.1505079904.7e482e79aa69d6f5-7a88ca087b2f1680 version[0,1] not find in stale rs version map
W1103 16:04:43.336555 17156 tablet.cpp:541] delete stale rowset tablet=49090.1505079904.7c43acdaffbeaa70-7d2ffb402f6aaf87 version[0,1] not find in stale rs version map
W1103 16:04:43.337074 17156 tablet.cpp:541] delete stale rowset tablet=49074.1505079904.b84d834d3508c0c8-67f9f9453cb79bab version[0,1] not find in stale rs version map
W1103 19:30:37.957989 17087 fragment_mgr.cpp:241] Got error while opening fragment 45942c99b7f544a3-99c667b2913c261d: Cancelled: Cancelled
W1204 12:10:14.370339 17250 task_worker_pool.cpp:1232] report version 16666583292462 change to 16666583292465
W1221 01:10:14.432669 17250 task_worker_pool.cpp:1232] report version 16666583482387 change to 16666583482391
W1224 05:00:14.652438 17250 task_worker_pool.cpp:1232] report version 16666583518806 change to 16666583518815
W1229 14:43:39.616370 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.627974 17202 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.628669 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.629312 17203 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.630019 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.630700 17203 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.631330 17202 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.631743 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.632444 17202 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.632879 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1230 11:37:40.934886 17248 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 11:37:40.961969 17248 utils.cpp:108] fail to get master client from cache. host=10.55.1.106, port=9020, code=THRIFT_RPC_ERROR
W1230 11:37:40.962019 17248 task_worker_pool.cpp:1598] report TASK failed. status: 0, master host: 10.55.1.106, port:9020
W1230 11:37:52.962467 17248 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 11:37:52.963227 17248 utils.cpp:108] fail to get master client from cache. host=10.55.1.106, port=9020, code=THRIFT_RPC_ERROR
W1230 11:37:52.963281 17248 task_worker_pool.cpp:1598] report TASK failed. status: 0, master host: 10.55.1.106, port:9020
W1230 11:38:04.029600 17249 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 11:38:06.963783 17248 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 11:38:08.629869 17250 utils.cpp:104] master client, retry finishTask: write() send(): Broken pipe
W1230 11:38:15.543959 17248 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 11:40:02.037972 17310 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 11:40:03.066108 17310 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 11:40:04.200903 17310 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 12:20:02.506862 17299 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 12:20:03.511305 17299 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 12:30:02.390944 17308 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 12:30:03.394768 17308 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 12:40:02.405933 17307 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 12:40:03.409085 17307 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:05:02.420872 17294 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:05:03.423832 17294 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:10:02.383661 17301 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:10:03.386349 17301 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:15:02.419265 17308 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:15:03.422350 17308 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:20:02.409471 17298 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:20:03.413426 17298 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:35:02.375789 17310 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:35:03.380138 17310 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:40:02.418221 17300 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:40:03.422461 17300 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 14:30:02.387089 17298 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 14:30:03.392902 17298 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 15:05:02.272935 27955 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 15:05:03.277900 27955 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 15:07:33.570282 27910 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 15:07:33.570978 27910 utils.cpp:108] fail to get master client from cache. host=10.55.1.106, port=9020, code=THRIFT_RPC_ERROR
W1230 15:07:33.571022 27910 task_worker_pool.cpp:1598] report TASK failed. status: 0, master host: 10.55.1.106, port:9020
W1230 15:07:45.571471 27910 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 15:07:45.572196 27910 utils.cpp:108] fail to get master client from cache. host=10.55.1.106, port=9020, code=THRIFT_RPC_ERROR
W1230 15:07:45.572248 27910 task_worker_pool.cpp:1598] report TASK failed. status: 0, master host: 10.55.1.106, port:9020
W1230 15:07:47.444311 27911 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 15:07:47.444708 27911 utils.cpp:108] fail to get master client from cache. host=10.55.1.106, port=9020, code=THRIFT_RPC_ERROR
W1230 15:07:47.444742 27911 task_worker_pool.cpp:1598] report DISK failed. status: 0, master host: 10.55.1.106, port:9020
W1230 15:08:20.749279 27910 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 15:08:20.750237 27910 utils.cpp:108] fail to get master client from cache. host=10.55.1.106, port=9020, code=THRIFT_RPC_ERROR
W1230 15:08:20.750298 27910 task_worker_pool.cpp:1598] report TASK failed. status: 0, master host: 10.55.1.106, port:9020
W1230 15:08:35.750669 27910 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 15:10:02.436517 27959 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 15:10:03.448431 27959 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 15:10:04.453572 27959 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 15:50:59.024062 27953 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 15:51:00.028187 27953 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.W1102 15:18:44.785388 17134 fragment_mgr.cpp:241] Got error while opening fragment 254b3a6c50de4361-97f4a9f86f262ef2: Cancelled: Cancelled
W1102 16:46:47.238984 17109 vdata_stream_sender.h:249] failed to send brpc batch, error=RPC call is timed out, error_text=[E1008]Reached timeout=300000ms @10.55.1.108:8060, client: 10.55.1.107
W1102 16:46:47.239799 17109 fragment_mgr.cpp:241] Got error while opening fragment 20526df6a1394842-92b4b7a07f0b86b6: Thrift rpc error: failed to send brpc batch, error=RPC call is timed out, error_text=[E1008]Reached timeout=300000ms @10.55.1.108:8060, client: 10.55.1.107
W1102 16:46:47.254287 17109 vdata_stream_sender.h:249] failed to send brpc batch, error=RPC call is timed out, error_text=[E1008]Reached timeout=300000ms @10.55.1.108:8060, client: 10.55.1.107
W1102 16:46:47.254333 17109 vdata_stream_sender.h:249] failed to send brpc batch, error=RPC call is timed out, error_text=[E1008]Reached timeout=300000ms @10.55.1.108:8060, client: 10.55.1.107
W1103 15:30:01.428149 17203 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.429009 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.429538 17203 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.430187 17202 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.430976 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.431612 17202 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.432267 17203 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.433130 17202 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.433645 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 15:30:01.434274 17203 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1667460601, base_tablet_create_time=1667460601
W1103 16:01:42.258661 17156 tablet.cpp:541] delete stale rowset tablet=49082.1505079904.314f77fcf9c8a8bc-a8904b682d00949d version[0,1] not find in stale rs version map
W1103 16:04:43.331810 17156 tablet.cpp:541] delete stale rowset tablet=49078.1505079904.97469d0d0e1463c9-7c82eddec3a58fa7 version[0,1] not find in stale rs version map
W1103 16:04:43.332846 17156 tablet.cpp:541] delete stale rowset tablet=49086.1505079904.d14c77805f0094fd-8cace9c0e85f0c8e version[0,1] not find in stale rs version map
W1103 16:04:43.334069 17156 tablet.cpp:541] delete stale rowset tablet=49110.1505079904.bc4062d1c6bb7cc3-3f769aafca3c6bbe version[0,1] not find in stale rs version map
W1103 16:04:43.334560 17156 tablet.cpp:541] delete stale rowset tablet=49106.1505079904.dc497bf3d4c5746b-5641ff032295a6bf version[0,1] not find in stale rs version map
W1103 16:04:43.335085 17156 tablet.cpp:541] delete stale rowset tablet=49102.1505079904.2247653a206d9982-eb0264d8be654990 version[0,1] not find in stale rs version map
W1103 16:04:43.335626 17156 tablet.cpp:541] delete stale rowset tablet=49098.1505079904.2a44367aa101c75a-ebbc78aa590b04b3 version[0,1] not find in stale rs version map
W1103 16:04:43.336102 17156 tablet.cpp:541] delete stale rowset tablet=49094.1505079904.7e482e79aa69d6f5-7a88ca087b2f1680 version[0,1] not find in stale rs version map
W1103 16:04:43.336555 17156 tablet.cpp:541] delete stale rowset tablet=49090.1505079904.7c43acdaffbeaa70-7d2ffb402f6aaf87 version[0,1] not find in stale rs version map
W1103 16:04:43.337074 17156 tablet.cpp:541] delete stale rowset tablet=49074.1505079904.b84d834d3508c0c8-67f9f9453cb79bab version[0,1] not find in stale rs version map
W1103 19:30:37.957989 17087 fragment_mgr.cpp:241] Got error while opening fragment 45942c99b7f544a3-99c667b2913c261d: Cancelled: Cancelled
W1204 12:10:14.370339 17250 task_worker_pool.cpp:1232] report version 16666583292462 change to 16666583292465
W1221 01:10:14.432669 17250 task_worker_pool.cpp:1232] report version 16666583482387 change to 16666583482391
W1224 05:00:14.652438 17250 task_worker_pool.cpp:1232] report version 16666583518806 change to 16666583518815
W1229 14:43:39.616370 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.627974 17202 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.628669 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.629312 17203 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.630019 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.630700 17203 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.631330 17202 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.631743 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.632444 17202 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1229 14:43:39.632879 17204 tablet_manager.cpp:354] new tablet's create time is less than or equal to old tabletnew_tablet_create_time=1672296219, base_tablet_create_time=1672296219
W1230 11:37:40.934886 17248 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 11:37:40.961969 17248 utils.cpp:108] fail to get master client from cache. host=10.55.1.106, port=9020, code=THRIFT_RPC_ERROR
W1230 11:37:40.962019 17248 task_worker_pool.cpp:1598] report TASK failed. status: 0, master host: 10.55.1.106, port:9020
W1230 11:37:52.962467 17248 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 11:37:52.963227 17248 utils.cpp:108] fail to get master client from cache. host=10.55.1.106, port=9020, code=THRIFT_RPC_ERROR
W1230 11:37:52.963281 17248 task_worker_pool.cpp:1598] report TASK failed. status: 0, master host: 10.55.1.106, port:9020
W1230 11:38:04.029600 17249 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 11:38:06.963783 17248 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 11:38:08.629869 17250 utils.cpp:104] master client, retry finishTask: write() send(): Broken pipe
W1230 11:38:15.543959 17248 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 11:40:02.037972 17310 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 11:40:03.066108 17310 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 11:40:04.200903 17310 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 12:20:02.506862 17299 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 12:20:03.511305 17299 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 12:30:02.390944 17308 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 12:30:03.394768 17308 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 12:40:02.405933 17307 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 12:40:03.409085 17307 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:05:02.420872 17294 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:05:03.423832 17294 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:10:02.383661 17301 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:10:03.386349 17301 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:15:02.419265 17308 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:15:03.422350 17308 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:20:02.409471 17298 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:20:03.413426 17298 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:35:02.375789 17310 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:35:03.380138 17310 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:40:02.418221 17300 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 13:40:03.422461 17300 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 14:30:02.387089 17298 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 14:30:03.392902 17298 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 15:05:02.272935 27955 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 15:05:03.277900 27955 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 15:07:33.570282 27910 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 15:07:33.570978 27910 utils.cpp:108] fail to get master client from cache. host=10.55.1.106, port=9020, code=THRIFT_RPC_ERROR
W1230 15:07:33.571022 27910 task_worker_pool.cpp:1598] report TASK failed. status: 0, master host: 10.55.1.106, port:9020
W1230 15:07:45.571471 27910 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 15:07:45.572196 27910 utils.cpp:108] fail to get master client from cache. host=10.55.1.106, port=9020, code=THRIFT_RPC_ERROR
W1230 15:07:45.572248 27910 task_worker_pool.cpp:1598] report TASK failed. status: 0, master host: 10.55.1.106, port:9020
W1230 15:07:47.444311 27911 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 15:07:47.444708 27911 utils.cpp:108] fail to get master client from cache. host=10.55.1.106, port=9020, code=THRIFT_RPC_ERROR
W1230 15:07:47.444742 27911 task_worker_pool.cpp:1598] report DISK failed. status: 0, master host: 10.55.1.106, port:9020
W1230 15:08:20.749279 27910 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 15:08:20.750237 27910 utils.cpp:108] fail to get master client from cache. host=10.55.1.106, port=9020, code=THRIFT_RPC_ERROR
W1230 15:08:20.750298 27910 task_worker_pool.cpp:1598] report TASK failed. status: 0, master host: 10.55.1.106, port:9020
W1230 15:08:35.750669 27910 utils.cpp:104] master client, retry finishTask: No more data to read.
W1230 15:10:02.436517 27959 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 15:10:03.448431 27959 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 15:10:04.453572 27959 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 15:50:59.024062 27953 thrift_rpc_helper.cpp:63] retrying call frontend service after 1000 ms, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.
W1230 15:51:00.028187 27953 thrift_rpc_helper.cpp:76] call frontend service failed, address=TNetworkAddress(hostname=10.55.1.106, port=9020), reason=No more data to read.

**Comment by LemonLiTree** [comment]
升级到1.1.5看看呢
